hivemq-edge/src/main/java/com/hivemq/api/resources/impl/ProtocolAdaptersResourceImpl.java

```diff
@@ -101,6 +99,7 @@
 public class ProtocolAdaptersResourceImpl extends AbstractApi implements ProtocolAdaptersApi {
 
     private static final @NotNull Logger log = LoggerFactory.getLogger(ProtocolAdaptersResourceImpl.class);
+    public static final String INTERNAL_ERROR_PLEASE_CHECK_LOGS = "Internal error, please check logs.";
```

Comment:
needs to be a json.

Reply:
Wasn't JSON before.
As agreed with Nicolas I am not changing the returned type, instead returning what we had befire and sticking to a simple error message until we rework all of them.